src/all/everiaclubcom/src/eu/kanade/tachiyomi/extension/all/everiaclubcom/EveriaClubCom.kt

```diff
@@ -0,0 +1,149 @@
+package eu.kanade.tachiyomi.extension.all.everiaclubcom
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.Filter
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.HttpSource
+import eu.kanade.tachiyomi.util.asJsoup
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import okhttp3.Response
+import org.jsoup.nodes.Element
+import rx.Observable
+
+class EveriaClubCom() : HttpSource() {
+    override val baseUrl = "https://www.everiaclub.com"
+    override val lang = "all"
+    override val name = "EveriaClub (unoriginal)"
+    override val supportsLatest = true
+
+    override fun headersBuilder() = super.headersBuilder()
+        .add("Referer", "$baseUrl/")
+
+    private val Element.imgSrc: String
+        get() = attr("data-original")
+            .ifEmpty { attr("data-lazy-src") }
+            .ifEmpty { attr("data-src") }
+            .ifEmpty { attr("src") }
```

Comment:
wouldn't checking with `hasAttr` be better?

Reply:
like this?
```
    private val Element.imgSrc: String
        get() = when {
            hasAttr("data-original") -> attr("data-original")
            hasAttr("data-lazy-src") -> attr("data-lazy-src")
            hasAttr("data-src") -> attr("data-src")
            hasAttr("src") -> attr("src")
            else -> ""
        }
        ```

Reply:
sure, and make nullable so else returns null